### PR TITLE
Add English docs

### PR DESCRIPTION
**Pull request by k0ngk0ng**
1. Add English version of  `deploy_baremetal.md` and `bgp_config.md` 
2. The configuration of other routers will be written in `router_config.md` 
3. This doc contains the usage of eip range

Pls review this doc, thanks.

### CODE REVIEW

File: doc/deploy_baremetal.md [hunk 1]
```diff
@@ -83,8 +89,20 @@
...
     EOF
     ```
+    Sample: [Eip](https://github.com/kubesphere/porter/blob/master/test/samples/eip.yaml)   
 
-2. 部署测试Service. Service必须要添加如下一个annotations，type也要指定为LoadBalancer,如下：
+    **Note: Eip address now supports 3 types:**
+   
+    - IP Address         
+        `192.168.0.1`
+    - IP Network/CIDR
```
Comment (FeynmanZhou) [inline doc/deploy_baremetal.md:98]: I guess `IP Network => IP Network segment` would be better

File: doc/deploy_baremetal.md [hunk 2]
```diff
@@ -1,49 +1,54 @@
...
+
+## Prerequisites
+1.  Your router needs to support the BGP protocol
+2.  Your router needs to support Equal-cost multi-path routing (ECMP) if you want to enable load-balancing on the router. Including the following features:
+    - Support multi-path routing
+    - Support BGP Additional-Paths
+3. If there is a router that does not support the BGP protocol (or is not allowed to enable the BGP protocol), you need to manually write the nexthop route of EIP on this router (or use other routing protocols)
+
+## Install Porter
+ 1. Install Kubernetes Cluster
```
Comment (FeynmanZhou) [inline doc/deploy_baremetal.md:13]: It seems a Kubernetes cluster should listed in Prerequisite

File: doc/deploy_baremetal.md [hunk 3]
```diff
@@ -70,8 +75,9 @@
         Graceful Restart capability: advertised
     ```
 
-## 部署示例
-1.  添加一个EIP池（单个eip请添加/32）
+## Deployment
+1.  Add an Eip resource
```
Comment (FeynmanZhou) [inline doc/deploy_baremetal.md:79]: an Eip resource = an Eip pool?

File: doc/deploy_baremetal.md [hunk 4]
```diff
@@ -103,22 +120,24 @@
...
+ 
     10.11.11.11/32, ubest/mbest: 3/0
     *via 10.10.12.2, [20/0], 00:03:38, bgp-65001, external, tag 65000
     *via 10.10.12.3, [20/0], 00:03:38, bgp-65001, external, tag 65000
     *via 10.10.12.4, [20/0], 00:03:38, bgp-65001, external, tag 65000
 
     ```
-4. 在集群上使用`kubectl get eip`观察当前集群eip的使用情况
\ No newline at end of file
+4. Use `kubectl get eip` to watch the current usage of Eip
```
Comment (FeynmanZhou) [inline doc/deploy_baremetal.md:143]: Eip => EIP

Comment (k0ngk0ng) [reply doc/deploy_baremetal.md]: Done.

**Pull request merged**